smallboard/settings.py

```diff
@@ -168,6 +168,16 @@
 DATABASES["default"] = dj_database_url.config(conn_max_age=600, ssl_require=False)
 DATABASES["default"]["TEST"] = {"NAME": "test_smallboard"}
 
+# Discord API. See
+# https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-
+DISCORD_API_TOKEN = os.environ.get("DISCORD_API_TOKEN", None)
+
+# Discord server ID.
+DISCORD_GUILD_ID = os.environ.get("DISCORD_GUILD_ID", None)
+
+# Category (folder) to contain generated channels.
+DISCORD_GUILD_CATEGORY = os.environ.get("DISCORD_GUILD_CATEGORY", "puzzles")
```

Comment:
I wonder if we should also shard this (in addition to archive). Feel free to do this in a feature PR.

Reply:
Yeah +1 we should probably implement some sharding scheme around this. Created #267 to track.